streampipes-data-explorer/src/main/java/org/apache/streampipes/dataexplorer/query/DataExplorerQueryExecutor.java

```diff
@@ -53,23 +54,23 @@ public DataExplorerQueryExecutor(int maximumAmountOfEvents) {
   public SpQueryResult executeQuery(SelectQueryParams params,
```

Comment:
It looks likte the method `getAmountOfResults` is not used anymore and can be removed.